packages/commons/src/Utility.ts

```diff
@@ -68,7 +68,7 @@ export class Utility {
    *
    * The method also flips the cold start status to `false` after the first invocation.
    */
-  public getColdStart(): boolean {
+  protected getColdStart(): boolean {
```

Comment:
I'm writing this here because these are the lines that changed, but you need to update the docstring for this utility.

Also, I probably don't know where this is in the codebase, and I'm 100% sure you have a way to do this, but it seems like you're missing a method to just get the current status of the `coldStart` variable without changing it, no?

Reply:
Ohh you're right! I saw now this is protected. Great.